### PR TITLE
Clearing data list

### PR DESCRIPTION
**Pull request by mokaithered**
Hi all,

I was playing around with gantt charts and experience that the legend (colorbar) needs to be scrolled to see my defined 'Tasks' at the very bottom (which is actually visible at the examples as well, above the 'Tasks' are hidden traces, which are even clickable). After checking through _gantt.py I identified the following rows, where the variable 'data' has many entries. 

new lines @ 277 
new lines @ 359
new lines @ 517

Can someone please verify and if agreeing, pushing the change to the repo?

Many thanks
Cheers
J

---edit---
found some issue and had to fix this

### CODE REVIEW

**Comment by jackparmer** [comment]
@mokaithered I'm trying to replicate this legend issue with the hidden traces, but can't find an example, can you please drop a screenshot or GIF here?

File: plotly/figure_factory/_gantt.py [hunk 1]
```diff
@@ -274,7 +274,10 @@ def gantt_colorscale(chart, colors, title, index_col, show_colorbar, bar_width,
 
         if show_colorbar is True:
             # generate dummy data for colorscale visibility
-            data.append(
+            datalisted = data
```
[the remark this answers is not in the document]
Comment (mokaithered) [reply plotly/figure_factory/_gantt.py]: sure, underscores or none or naming conventions, I am good with any.

**Comment by mokaithered** [comment]
grafik

I hope this helps! Sorry for the grey areas, I need to be a bit careful concerning the data.

**Comment by jonmmease** [comment]
Extra space in Gantt-chart legend issue was resolved by https://github.com/plotly/plotly.py/pull/1110

**Pull request closed without merging**